### ISSUE TITLE
Windows high contrast mode issue

### ISSUE THREAD

**Issue by helennickols** [ministryofjustice/moj-frontend#2001]
## What

There is an issue with windows high contrast mode where the focus state of the side navigation is not visible. This came up in the a11y review but it's a usability issue.  See [Do issues with Windows high contrast mode fall under WCAG 2.1? #623](https://github.com/w3c/wcag/issues/623#issuecomment-865916948).

In the image below the focus is on Patterns. 

<img width="649" height="865" alt="Image" src="https://github.com/user-attachments/assets/9ad63b1c-ca21-4ab5-b424-3a0066e7e20d" />

This was put forward by Eu on 13 Oct 2025. 

## Why

Meet accessibility guidelines.

## Who needs to be involved

- Interaction Designer
- Content Designer
- Developer
- Accessibility Specialist

## Done

Change has been made.

**Comment by Asma-Ban** [comment]
Small fix, could move to next cycle if there is little time.

**Comment by helennickols** [comment]
I've removed myself from this ticket because it's a dev task.
